assignments/callbacks.js

```diff
@@ -24,32 +24,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
```

Comment:
callbacks refer to another function to callback and uses that information. For instance the get length function with a callback could look like this.
function getLength(arr, cb) {
  // getLength passes the length of the array into the callback.
  console.log(cb(arr));
}

let arrayLength = function(items) {
  return items.length;
}

getLength(items,arrayLength);